scripts/upload-artifacts

```diff
@@ -20,12 +20,15 @@ if TAG=$(git describe --exact-match --tags 2>/dev/null); then
     if [[ -z $GITHUB_TOKEN ]]; then
         echo GITHUB_TOKEN not set, skipping artifact upload to GitHub
     else
+        UPLOAD_URL=$(curl -sfL https://api.github.com/repos/cri-o/cri-o/releases |
+            jq -r '.[] | select(.tag_name == "v1.18.0") | .upload_url')
```

Comment:
Any chance we could read the "v1.18.0" from somewhere?  Just trying to avoid having to remember to come back and bump this.

Reply:
Oops that’s a typo. It should be the last tag, thanks for catching that! 👍

Reply:
Fixed it.